docs/toc.md

```diff
@@ -253,7 +253,7 @@
 ### [Basic Types](csharp/basic-types.md)
 ### [Classes](csharp/classes.md)
 ### [Structs](csharp/structs.md)
-### [🔧 Tuples and unnamed types](csharp/tuples.md)
+### [Tuples and unnamed types](csharp/tuples.md)
```

Comment:
The article does not mention "unnamed types", so I think the TOC shouldn't either.